.circleci/config.yml

```diff
@@ -42,9 +47,7 @@ jobs:
       - ios/install-dependencies:
           bundle-install: true
           pod-install: true
-      - run:
-          name: Copy demo SPCredentials
-          command: mkdir -p Simplenote/Credentials && cp Simplenote/SPCredentials-demo.swift Simplenote/Credentials/SPCredentials.swift
+      - copy_secrets
```

Comment:
It was neat to copy the demo credentials over, but now that we fully rely on the aggregate target build phase, this is not sufficient anymore.